auto_backup/models/db_backup.py

```diff
@@ -252,7 +253,8 @@ def schedule_backup(self):
                     # Close the SFTP session.
                     sftp.close()
                 except Exception as e:
-                    _logger.debug('Exception! We couldn\'t back up to the FTP server..')
+                    sftp.close()
+                    _logger.error('Exception! We couldn\'t back up to the FTP server..')
```

Comment:
👍 Indeed, this should have been an `error` instead of a `debug` in the first place.